doc/select.md

```diff
@@ -0,0 +1,84 @@
+# Select examples
```

Comment:
It would be grate to say somewhere if we specify sharding key (in our case it's primary key always) or bucket_id (https://github.com/tarantool/crud/commit/d7d45af3d962a8c347f63c4253742c211f851bcf) select will be performed on single node otherwise map-reduce over all nodes will be occurred.